maps/away/ships/idris/idris_cruiser.dm

```diff
@@ -0,0 +1,102 @@
+/datum/map_template/ruin/away_site/idris_cruiser
+	name = "Idris Cruiser"
+	description = "A small luxury cruiser run by Idris Incorporated's subsidiary, Celestial Cruises. The Argentum-class is more of a yacht than a proper cruise ship, and is easily dwarfed by the fleet's larger vessels. However, it makes up for its diminuitive size by its speed, flexibility, and low maintenance cost. It adopts a unique wandering business model, where it roams the Spur and caters to tired traveling vessel crews seeking a getaway among the stars. It comes with a bar and restaurant, a pool, a spa, and a viewing lounge, as well as four suites for overnight stayers."
+	suffixes = list("ships/idris/idris_cruiser.dmm")
+	sectors = list(ALL_CORPORATE_SECTORS)
+	spawn_weight = 1
+	ship_cost = 1
+	id = "idris_cruiser"
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/idris_cruiser_shuttle)
+	unit_test_groups = list(3)
+
+/singleton/submap_archetype/idris_cruiser
+	map = "Idris Cruiser"
+	descriptor = "A small luxury cruiser run by Idris Incorporated's subsidiary, Celestial Cruises. The Argentum-class is more of a yacht than a proper cruise ship, and is easily dwarfed by the fleet's larger vessels. However, it makes up for its diminuitive size by its speed, flexibility, and low maintenance cost. It adopts a unique wandering business model, where it roams the Spur and caters to tired traveling vessel crews seeking a getaway among the stars. It comes with a bar and restaurant, a pool, a spa, and a viewing lounge, as well as four suites for overnight stayers."
+
+// ship
+
+/obj/effect/overmap/visitable/ship/idris_cruiser
+	name = "Idris Cruiser"
+	class = "IIV"
+	desc = "A small luxury cruiser run by Idris Incorporated's subsidiary, Celestial Cruises. The Argentum-class is more of a yacht than a proper cruise ship, and is easily dwarfed by the fleet's larger vessels. However, it makes up for its diminuitive size by its speed, flexibility, and low maintenance cost. It adopts a unique wandering business model, where it roams the Spur and caters to tired traveling vessel crews seeking a getaway among the stars. It comes with a bar and restaurant, a pool, a spa, and a viewing lounge, as well as four suites for overnight stayers."
+	icon_state = "sanctuary"
+	moving_state = "sanctuary_moving"
+	colors = "#5acfc0"
+	designer = "Idris Incorporated - Celestial Cruises"
+	volume = "82 meters length, 60 meters beam/width, 28 meters vertical height"
+	drive = "Warp Capable, Hybrid Phoron Bluespace Drive"
```

Comment:
Very minor lore thing, but IIRC warp/bluespace hybrid drives such as the Horizon's are very rare and experimental, and I'm not sure it would fit for just an Idris cruise ship.

Reply:
fixed this